Final/SyncLinkedList.cpp

```diff
@@ -0,0 +1,32 @@
+#include "SyncLinkedList.h"
+
+// Constructor
+SyncLinkedList::SyncLinkedList() {
+    linked_list = new LinkedList();
+}
+// Destructor
+SyncLinkedList::~SyncLinkedList() {
+    linked_list->~LinkedList();
```

Comment:
in general, you never need to call the destructor. it'll be called automatically when the variable is deallocated. since you used `new` to create the LinkedList, you'll need to `free` the linked list here.